src/plots/cartesian/axes.js

```diff
@@ -2867,7 +2882,7 @@ axes.drawLabels = function(gd, ax, opts) {
             } else {
                 var mjWidth = Drawing.bBox(mathjaxGroup.node()).width;
                 var mjShift = mjWidth * {end: -0.5, start: 0.5}[anchor];
-                mathjaxGroup.attr('transform', transform + (mjShift ? 'translate(' + mjShift + ',0)' : ''));
+                mathjaxGroup.attr('transform', transform + (mjShift ? strTranslate(mjShift, 0) : ''));
```

Comment:
you don't need `mjShift ?`, `strTranslate` already does that :)

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/5254/commits/15c4fa02bde0db678dcd4637a9c228df55c9a6d3.